### PR TITLE
3796 Add case deadline validation to the get all case deadlines interactor

### PR DESCRIPTION
**Pull request by ericsorenson**

### CODE REVIEW

File: docs/entities/Case.md [hunk 1]
```diff
@@ -3,10 +3,37 @@
 ### associatedJudge
 
 
-Defaults to Chief Judge.
+Judge assigned to this Case. Defaults to Chief Judge.
 
 > `string` | required
 
+### automaticBlocked
```
Comment (matthopson) [inline docs/entities/Case.md:10]: `automatic` is an adjective, not an adverb. Maybe we should think different about this.

**Review by matthopson — APPROVED** [review]

File: docs/entities/Case.md (hunk 1 above)
Comment (kkoskelin) [reply docs/entities/Case.md:10]: very clever, young man.

File: shared/src/business/entities/CaseDeadline.js [hunk 2]
```diff
@@ -37,31 +39,40 @@ CaseDeadline.VALIDATION_ERROR_MESSAGES = {
...
-    .required(),
+    .required()
+    .description('When the Case Deadline expires.'),
   description: joi
     .string()
     .max(120)
-    .required(),
+    .min(1)
+    .required()
+    .description('User provided dscription of the Case Deadline.'),
```
Comment (kkoskelin) [inline shared/src/business/entities/CaseDeadline.js]: `description`

File: docs/entities/Case.md (hunk 1 above)
Comment (rachaelparris) [reply docs/entities/Case.md:10]: Hey, naming thing is hard.

**Review by kkoskelin — APPROVED** [review]
after typo is fixed, 👍

File: docs/entities/Case.md (hunk 1 above)
Comment (matthopson) [reply docs/entities/Case.md:10]: 😂

**Review by rachaelparris — APPROVED** [review]

**Review by wbyoko — COMMENTED** [review]
This seems to filter out the case title.

Screenshot 2020-01-31 14 16 10

**Pull request merged**